### DISCUSSION TITLE
process ENVISAT ASAR by stackStripmap.py

### DISCUSSION THREAD

**Discussion by zhouhuayun** [isce-framework/isce2#489] [Q&A]
Hello,
when I run this command  stackStripMap.py -s */slc/ --bbox '*' -w ./ -d /*.dem.wgs84 -m 20081023 -W interferogram -t 1000 -b 500 -a 5 -r 1 -u snaphu --nofocus -useGPU --zero -f 0.5 
this error occured: dbm.error: db file doesn't exist; use 'c' or 'n' flag to create a new db
and this is the screenshot：
![screenshot](https://user-images.githubusercontent.com/42872898/168418808-455690b0-7e3f-421d-b713-d1a8a65970cd.png)

**Comment by pburgi-usgs** [comment]
@zhouhuayun, were you ever able to find a solution to this error? I am running into the same problem, except with RADARSAT-2 data. I had another error in the earlier in the same script, which I was able to fix by removing the .bak, .dat, and .dir files from the unpacked SAR image directories (solution described [here](https://github.com/isce-framework/isce2/discussions/365)). To try to fix this problem, I changed the problem command in stackStripMap.py:  

```
pairs = selectPairs(inps,stackReferenceDate, secondaryDates, acquisitionDates,doBaselines=True)
```
such that I set the "doBaselines" argument to "False". 

That allowed stackStripMap.py to finish running, BUT it did not actually fix the problem. Running the run files still failed since it does need the baseline step. Any guidance would be appreciated!
